### DISCUSSION TITLE
Why should I choose Dokku over CapRover?

### DISCUSSION THREAD

**Discussion by kamzata** [dokku/dokku#4939] [Q&A]
I installed Dokku and I find it nice to use. Then I found out CapRover which, for what I can understand, gives more features: e.g. the additional possibility to scale my application on more servers and the nice UI to handle all the things. However, CapRover development seems less active than Dokku's one.

My question is: why should I choose Dokku over CapRover, and why should I choose CapRover over Dokku?

**Comment by alhafoudh** [comment]
LTIC CapRover does not build images auto-magically like dokku does (using gliderlabs/herokuish docker image).
